### PR TITLE
[UI-Fix] Group Chat in ChatList Rendering

### PR DESCRIPTION
**Pull request by vineetvk01**
**UI-Issue:**
Currently in IGdm we don't show difference between a normal chat and a group chat in the chat-List.

**Fix**
We can show a group chat of any number of members with any 5 user's images in chat list..

**Preview:**
rendering-grou-chat

### CODE REVIEW

File: browser/js/funcs.js [hunk 1]
```diff
@@ -40,8 +40,20 @@ function getChatTitle (chat_) {
...
+  }
+  return false;
+}
+
 function getChatThumbnail (chat_) {
-  return chat_.accounts[0]._params.picture;
+  if (chat_.accounts[0] && !isGroupChat(chat_)) {
+    return chat_.accounts[0]._params.picture;
+  }
+  return chat_.accounts.map((account)=>{
```
Comment (ifedapoolarewaju) [inline browser/js/funcs.js]: can we set this [eslint rule](https://eslint.org/docs/rules/arrow-spacing), so it enforces that we always have spaces in arrow functions. I think it makes the codebase neater if we have it that way.

File: browser/js/renderers.js [hunk 2]
```diff
@@ -250,13 +250,26 @@ function renderChatContextMenu (chatId, li) {
   menu.popup({});
 }
 
+function createThumbnailDom (imageUrls) {
+  if (typeof imageUrls === 'string') {
+    return dom(`<div><img class="thumb" src="${imageUrls}"></div>`); 
+  }
+  let html = '<div>';
+  imageUrls.forEach((imageUrl, index)=>{
+    if (index < 5) html += `<img class="thumb ${ index === 0 ? '' : 'group'}" src="${imageUrl}"></img>`;
```
Comment (ifedapoolarewaju) [inline browser/js/renderers.js]: I vote that we expand this`if` block into curly braces for readability.

File: browser/css/app.css [hunk 3]
```diff
@@ -58,6 +58,15 @@ button, button:focus {
   border-radius: 50%;
 }
 
+.group{
+  margin-left: -2.6rem;
+  transition: margin-left 1s;
+}
+
+.group:hover{
+  margin-left: -1.0rem;
```
Comment (ifedapoolarewaju) [inline browser/css/app.css]: how good does this look when we reduce the window width to the very minimum? Is there still enough room for this animation? 🤔

File: browser/js/funcs.js (hunk 1 above)
Comment (vineetvk01) [reply browser/js/funcs.js]: Yes, Adding this rule with PR :dango:

File: browser/css/app.css (hunk 3 above)
Comment (vineetvk01) [reply browser/css/app.css]: No, I added media query now. Thanks for updating with this possible scenario.

**Pull request merged**